src/main/java/Parser.java

```diff
@@ -0,0 +1,102 @@
+public class Parser {
+    public enum Command {
+        TODO, DEADLINE, EVENT, LIST, MARK, UNMARK, BYE, UNKNOWN
+    }
+
+    public static Command parseCommand(String userInput) {
```

Comment:
Possibly implement checking of userInput not being null to ensure proper error handling

Reply:
Thanks for the spot! I just checked if they are properly handled. Commands that are null or not in the list are defaulted to unknowns currently in the switch case, so there will be a feedback message stating that it is an unknown command :)